<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>智能化养猪系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>IP 地址：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>端口号：</string>
    </property>
   </widget>
   <widget class="IPCtrl" name="ipaddr" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPort">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConnect">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>296</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>打开</string>
     </property>
     <addaction name="actionLoadConfig"/>
    </widget>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>保存</string>
     </property>
     <addaction name="actionSaveConfig"/>
    </widget>
    <addaction name="menu_2"/>
    <addaction name="menu_3"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelpAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoadConfig">
   <property name="text">
    <string>配置文件...</string>
   </property>
  </action>
  <action name="actionSaveConfig">
   <property name="text">
    <string>配置文件...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="text">
    <string>关于...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IPCtrl</class>
   <extends>QWidget</extends>
   <header>ipctrl.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
